src/locale/locales/fr/messages.po

```diff
@@ -5387,7 +5378,7 @@ msgstr "Pour désactiver le 2FA par e-mail, veuillez vérifier votre accès à l
 
 #: src/components/dms/ReportConversationPrompt.tsx:20
 msgid "To report a conversation, please report one of its messages via the conversation screen. This lets our moderators understand the context of your issue."
-msgstr ""
+msgstr "Pour signaler une conversation, veuillez signaler un de ses messages via l’écran de conversation. Cela permettra à nos modérateurs de comprendre le contexte du problème."
```

Comment:
```suggestion
msgstr "Pour signaler une conversation, veuillez signaler un de ses messages via l’écran de conversation. Cela permettra à la modération de comprendre le contexte du problème."
```
Would it make sense to use _la langue épicène_ here, using a phrasing something like this?

Reply:
Good catch again — yeah, I tried to never use _"modérateurs"_ for this exact reason. 👍